WordPress/gutenberg
# [RNMobile][Embed block] Disable inline preview for specific WP embed providers

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3991





## Description


The inline preview for Pinterest URLs is not yet supported, however, it's currently being shown. This is caused because this provider is being rendered as a WordPress embed, in which inline previews are enabled independently to the provider. For this reason, we're adding a list of unsupported providers for the WordPress embeds that will disable it.

## How has this been tested?



1. Add an Embed block.
1. Use a Pinterest URL (example: https://www.pinterest.com/annsawesomepins/).
1. Observe that the inline preview is not shown.

## Screenshots 
N/A

## Types of changes




Bug fix

## Checklist:
